src/alire/alire-properties-from_toml.ads

```diff
@@ -26,7 +27,8 @@ package Alire.Properties.From_TOML with Preelaborate is
                           Notes,
                           Project_Files,
                           Website,
-                          Tags);
+                          Tags,
+                          Auto_GPR_With);
```

Comment:
No need to change but for future occasions: my OCD is appeased by having these enums in alphabetical order if there's no reason for other ordering :-)